test/system/test_tpm2_pcrextend.sh

```diff
@@ -55,26 +56,33 @@ for alg in `tpm2_pcrlist -s | cut -d\  -f 3-`; do
 
   hash=${alg_hashes[$alg]}
 
+  if [ ! -z  $digests ]; then
+      digests="$digests,"
+  fi
+
+  digests="$digests$alg=$hash"
```

Comment:
I think this test was missing testing individual hash specs, I think we would probably want to test that a single hash spec works, maybe test them individually here while building a larger digest spec list.

Reply:
@williamcroberts individual hash specs are tested, since the next line in the shell script does `tpm2_pcrextend 9:$alg=$hash`, or what did you mean?

Reply:
Wow that was right in front of me, I missed it. Nothing to do here then.